data/streams.js

```diff
@@ -0,0 +1,97 @@
+const fs = require('fs');
+const concat = require('concat');
+const csv = require('csvtojson'); 
+const program = require('commander');
+
+function reverse(str) {
+    console.log(str.split("").reverse().join(""));
+}
+
+function transform(str) {
+    console.log(str.toUpperCase());
+}
+
+function outputFile(filePath) {
+    const reader = fs.createReadStream(filePath, {encoding: 'utf-8'});
+    reader.on('readable', () => {
```

Comment:
Consider piping the read stream to process.stdout.
The idea behind this task is not simply use console.log :)